tutorials/introductory/usage.py

```diff
@@ -243,15 +243,15 @@ def my_plotter(ax, data1, data2, param_dict):
 my_plotter(ax, data1, data2, {'marker': 'x'})
 
 ###############################################################################
-# or if you wanted to have 2 sub-plots:
+# or if you wanted to have two sub-plots:
 
 fig, (ax1, ax2) = plt.subplots(1, 2)
 my_plotter(ax1, data1, data2, {'marker': 'x'})
 my_plotter(ax2, data3, data4, {'marker': 'o'})
 
 ###############################################################################
-# For these simple examples this style seems like overkill, however
-# once the graphs get slightly more complex it pays off.
+# For these simple examples, this may seem over-engineered; however,
```

Comment:
I'm a big fan of the "no just/no simple" doc writing advice - "For these examples, this"

Reply:
Reduced the language even more now!